src/Foundation/NSDate.cs

```diff
@@ -54,30 +72,21 @@ public partial class NSDate {
 			if (dt.Kind == DateTimeKind.Unspecified)
 				throw new ArgumentException ("DateTimeKind.Unspecified cannot be safely converted");
 
-			return FromTimeIntervalSinceReferenceDate ((dt.ToUniversalTime ().Ticks - NSDATE_TICKS) / (double) TimeSpan.TicksPerSecond);
-		}
-#else
-		public static implicit operator DateTime (NSDate d)
```

Comment:
Removing the implicit operators is a breaking change.

Reply:
@spouliot said we're no longer building XAMCORE_2_0

Reply:
Ah, I didn't realize these were only !XAMCORE_2_0.